locust/html.py

```diff
@@ -75,6 +75,7 @@ def get_html_report(environment, show_download_link=True):
         "per_class": get_ratio(environment.user_classes, user_spawned, False),
         "total": get_ratio(environment.user_classes, user_spawned, True),
     }
+    from .stats import PERCENTILES_TO_CHART
```

Comment:
group this import with the one on line 6

Reply:
Thank you for the comment. importing PERCENTILES_TO_CHART at 6  does not work. What I mean by that,
stats. PERCENTILES_TO_CHART default is [0.95, 0.5] as list in stats.py. When passing new value from locust execution file like stats. PERCENTILES_TO_CHART = [0.8,0.3] as list for instance, when PERCENTILES_TO_CHART was read from html.py file, default [0.95, 0.5] was used not new value [0.8,0.3].  But importing PERCENTILES_TO_CHART at line 78 which is right before PERCENTILES_TO_CHART is used, new value  [0.8,0.3] is used by html.py.  I think this is caused by pass by reference etc.. Actually, passing new value as PERCENTILES_TO_CHART[0] = 0.8 works, even if importing PERCENTILES_TO_CHART at line 6. I would like to suggest to use PERCENTILES_TO_CHART_1 and  PERCENTILES_TO_CHART_2 in stats.py not as list. This should work.I hope this explanation makes sense to you. If you have any suggestion, let me know.